db/model/room.js

```diff
@@ -144,9 +144,7 @@ module.exports = function room(seq, dataTypes) {
         }
 
         if (instance.changed('active')) {
-          if (instance.active) {
-            instance.attachBotsAndPublish();
-          }
```

Comment:
Is this a required change for using the new format/realtime app, or is it a fix to something that's been wrong all along?
This goes all the way back to when it was first added in https://github.com/salesforce/refocus/pull/939.

Reply:
This was implemented somewhere else, and I didn't realise. Reverted